gpt4all-api/README.md

```diff
@@ -4,8 +4,12 @@ for serving inference from GPT4All models. The API matches the OpenAI API spec.
 
 ## Tutorial
 
+Following tutorial assumes that you are checked out this repo and cd into it.
+
 ### Starting the app
 
+`cwd: gpt4all/gpt4all-api`
```

Comment:
Change "cdw" to "working directory", easier to understand. And only write the actual path itself in monospace